### PR TITLE
Run fuzzer without using make

### PR DESCRIPTION
**Pull request by kgpai**
This is a workaround to using make since currently it tends to rebuild code.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -241,7 +241,12 @@ jobs:
       - run:
           name: "Run Fuzzer Tests"
           command: |
-            make fuzzertest NUM_THREADS=10
+            _build/debug/velox/expression/tests/velox_expression_fuzzer_test \
```
Comment (mbasmanova) [inline .circleci/config.yml:244]: Perhaps, add a comment to explain this workaround.

File: .circleci/config.yml [hunk 2]
```diff
@@ -241,7 +241,14 @@ jobs:
       - run:
           name: "Run Fuzzer Tests"
           command: |
-            make fuzzertest NUM_THREADS=10
+            # Run fuzzer using the built executable - we do this instead of make
+            # since currently make fuzzer-test tends to rebuild the project.
```
Comment (mbasmanova) [inline .circleci/config.yml:245]: fuzzer-test -> fuzzertest ?

**Review by mbasmanova — APPROVED** [review]
Thanks.

File: .circleci/config.yml (hunk 2 above)
Comment (kgpai) [reply .circleci/config.yml:245]: Yes; wonder why unittest and fuzzertest dont have hyphens but header-fix, format-fix, header-check etc do .. 😄

**Pull request closed without merging**